src/components/forms/PlanForm/index.tsx

```diff
@@ -303,8 +321,12 @@ const PlanForm = (props: PlanFormProps) => {
                 }}
                 className={errors.interventionType ? 'form-control is-invalid' : 'form-control'}
               >
-                <option value={InterventionType.FI}>{FOCUS_INVESTIGATION}</option>
-                <option value={InterventionType.IRS}>{IRS_TITLE}</option>
+                {isPlanTypeEnabled(InterventionType.FI) && (
+                  <option value={InterventionType.FI}>{FOCUS_INVESTIGATION}</option>
+                )}
+                {isPlanTypeEnabled(InterventionType.IRS) && (
+                  <option value={InterventionType.IRS}>{IRS_TITLE}</option>
+                )}
```

Comment:
Do we need to cater for other plan types here?

Reply:
`MDA Point` plan type added